### PR TITLE
Correcting a typo

### PR DESCRIPTION
**Pull request by haroon-sheikh**

### CODE REVIEW

File: website/_includes/manual/working-with-parameters.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ## Working with parameters
 
-DbFit enables you to use Fixture symbols as global variables during test execution, to store or read intermediate results. The .NET syntax to access symbols (`>>parameter` to store a value and `<<parameter` to read the value) is supported in both .NET and Java versions. In addition, you can uset he `Set Parameter` command to explicitly seta parameter value to a string.
+DbFit enables you to use Fixture symbols as global variables during test execution, to store or read intermediate results. The .NET syntax to access symbols (`>>parameter` to store a value and `<<parameter` to read the value) is supported in both .NET and Java versions. In addition, you can use the `Set Parameter` command to explicitly seta parameter value to a string.
```
Comment (MMatten) [inline website/_includes/manual/working-with-parameters.md]: Is it worth also fixing `seta` too?

Comment (haroon-sheikh) [inline website/_includes/manual/working-with-parameters.md]: Done.

**Comment by MMatten** [comment]
@haroon-sheikh looks good to me. 

@javornikolov any comments? Should these commits be squashed into one?

**Comment by javornikolov** [comment]
Thank you @haroon-sheikh!

> @javornikolov any comments? Should these commits be squashed into one?

Yes, I believe it's better to squash. And maybe make it clear in the commit message that the typos are related to docs.

**Pull request closed without merging**